packages/utils/telemetry-utils/src/events.ts

```diff
@@ -8,9 +8,25 @@
 import { EventEmitter } from "events";
 import { ITelemetryLoggerExt } from "./telemetryTypes";
 
+/**
+ * TODO
+ *
+ * @public
+ */
 export const connectedEventName = "connected";
+
+/**
+ * TODO
+ *
+ * @public
+ */
 export const disconnectedEventName = "disconnected";
 
+/**
+ * TODO
+ *
+ * @public
+ */
 export function safeRaiseEvent(
```

Comment:
This whole file should move to core-utils and be internal.

Reply:
+1. Confirmed no uses in bohemia nor loopsdk for any of the exports here.

Reply:
All marked as internal, and I added a file-level comment indicating that the contents should move. Holding off on doing any moving now, as it will be much easier to make such breaking changes after we have started trimming internal API members.